### PR TITLE
fix for Javadoc file

### PR DESCRIPTION
**Pull request by aitorme**
This fix solves all the warnings while creating the Javadoc file in Java 7, which turn into errors with Java 8, impeding it from being built.

### CODE REVIEW

**Comment by psibre** [comment]
I think you still need to push the `revert` commit.

**Comment by psibre** [comment]
Pleas see #352 for an update.

**Comment by aitorme** [comment]
there are many more errors that only seem to crop up when running MaryTTS in OSX, I will keep fixing them and comment when everything will be fine on my machine

**Comment by psibre** [comment]
@aitorme Could you attach the full error output (as `output.txt` or similar -- note the extension)?

**Comment by aitorme** [comment]
[output.txt](https://github.com/marytts/marytts/files/3548/output.txt)
many errors, like these, are coming from Jampack

**Comment by psibre** [comment]
Jampack has been externalized and shouldn't be a problem an more. Please merge [my PR](/aitorme/marytts/pull/1).

**Comment by psibre** [comment]
Please also format the javadoc in [en.Preprocess.java](https://github.com/marytts/marytts/blob/3eb26432e360709d31064b74d51f55de8830ced3/marytts-languages/marytts-lang-en/src/main/java/marytts/language/en/Preprocess.java#L37-L71)

File: marytts-common/src/main/java/marytts/fst/AlignerTrainer.java [hunk 1]
```diff
@@ -44,32 +44,34 @@
  * The algorithm, in its essence, is implemented after a description of levenshtein distance as it can be found in Wikipedia (see
  * the linked revision):
  * 
- * http://en.wikipedia.org/w/index.php?title=Levenshtein_distance&oldid=349201802#Computing_Levenshtein_distance
+ * @see <a
+ *      href="http://en.wikipedia.org/w/index.php?title=Levenshtein_distance&oldid=349201802#Computing_Levenshtein_distance">Computing
+ *      Levenshtein distance</a>
  *
```
Comment (psibre) [inline marytts-common/src/main/java/marytts/fst/AlignerTrainer.java]: `@see` is not for external weblinks! Please refer to the [reference](http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html) and use the correct annotations.

**Pull request closed without merging**